### ISSUE TITLE
existing add-ons like Adblock plus has disappeared

### ISSUE THREAD

**Issue by satshaji** [mozilla-mobile/fenix#14563]
I had Adblock plus add-on installed especially to watch YouTube videos using Firefox. But after this new update, this add-on has disappeared.i tried to add it again, but not able to do so.

**Comment by jawz101** [comment]
This is not a necessary issue.  It is well known that add-on support is currently limited to the 9 add-ons they have made available.

The whole browser was rewritten from scratch in this last upgrade.  It's basically a completely new Firefox so Adblock Plus will eventually come back but just not yet.  Meanwhile, the built in Tracking Protection and the offered ad blockers that are currently available do as much or more for you in the meantime.


Feel free to close the issue

**Comment by liuche** [comment]
You can also follow #14034 for the next steps in add-on support in Firefox, or the [Mozilla Add-ons Blog](https://blog.mozilla.org/addons/category/mobile/).

**Comment by satshaji** [comment]
Thank you for the quick support. I installed the adblocker that was available by default and it works.  I never used ad blockers earlier in mobile and so tolerated the ads as they came by especially using YouTube. But recently, I came to know that adblocker can block YouTube ads. So, I installed Firefox and also Adblock extension. Everything was working fine until today.  I am new to Android, Firefox and ad Block. 

As you said, it's not an issue. Thanks again.
